resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -323,10 +323,7 @@
 }
 
 sanitize_outputs(
-    $gameTitle,
-    $consoleName,
     $richPresenceData,
-    $user,
```

Comment:
All of these (except `richPresenceData`) are passed to other widgets, which are responsible for their own escaping when necessary.